.github/workflows/test.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       with:
         ruby-version: 2.6
         bundler: 2
-    - run: bundle --version | grep "Bundler version 2\.\d+.\d+"
+    - run: bundle --version | grep "Bundler version 2\.\d\+.\d\+"
```

Comment:
This pattern doesn't work for me locally, so it seems unlikely to work on GHA:
`echo 2.2.32 | grep "2\.\d\+.\d\+"` => exit 1, no output